### PR TITLE
Nrcan master merge

### PR DESCRIPTION
**Pull request by ckirney**
Pull request overview
---------------------

Merging changes from nrcan branch into master.  These changes include:
-Revising and streamlining several NECB tests
-Adjusting VAV implementation in necb to accommodate E+ issues
-Removing all Canadian weather files except for 8 which represent different Canadian regions and climate zones
-Switching 8 Canadian weather locations from CWEC2016 to CWEC2020 data
-Sundry other changes

 - Fixes #ISSUENUMBERHERE (IF THIS IS A DEFECT)

### Pull Request Author

 - [x] Method changes or additions
 - [x] Data changes or additions
 - [x] Added tests for added methods
 - [ ] If methods have been deprecated, update rest of code to use the new methods
 - [ ] Documented new methods using [yard syntax](https://rubydoc.info/gems/yard/file/docs/GettingStarted.md)
 - [ ] Resolved yard documentation errors for new code (ran `bundle exec rake doc`)
 - [ ] Resolved rubocop syntax errors for new code (ran `bundle exec rake rubocop`)
 - [x] All new and existing tests passes
 - [ ] If the code adds new `require` statements, ensure these are in core ruby or add to the gemspec

### Review Checklist

This will not be exhaustively relevant to every PR.
 - [x] Perform a code review on GitHub
 - [x] All related changes have been implemented: method additions, changes, tests
 - [x] Check rubocop errors
 - [x] Check yard doc errors
 - [ ] If fixing a defect, verify by running develop branch and reproducing defect, then running PR and reproducing fix
 - [ ] If a new feature, test the new feature and try creative ways to break it
 - [x] CI status: all green or justified

### CODE REVIEW

File: lib/openstudio-standards/standards/necb/common/btap_data.rb [hunk 1]
```diff
@@ -499,7 +499,8 @@ def space_table(model, cost_result)
   def climate_data
     # Store Geography Data
     geography_data = {}
-    geography_data['location_necb_hdd'] = @standard.get_necb_hdd18(@model)
+    geography_data['location_necb_hdd'] = @standard.get_necb_hdd18(model: @model, necb_hdd: true)
```
Comment (mdahlhausen) [inline lib/openstudio-standards/standards/necb/common/btap_data.rb:502]: why both fields?

Comment (ckirney) [reply lib/openstudio-standards/standards/necb/common/btap_data.rb:502]: I should fix that.  I evidently didn't look closely enough at what was already there when I wrote that.

Comment (ckirney) [reply lib/openstudio-standards/standards/necb/common/btap_data.rb:502]: Thanks for pointing that out.  I removed the redundant hdd report.

File: test/necb/unit_tests/tests/test_necb_weather_hdd.rb [hunk 2]
```diff
@@ -0,0 +1,128 @@
+require_relative '../../../helpers/minitest_helper'
+require_relative '../../../helpers/create_doe_prototype_helper'
+require 'json'
+require_relative '../../../helpers/necb_helper'
+include(NecbHelper)
+
+# *** Needs a re-write to use std paths etc ***
+
+
+#This test will check that the ERVs are added and the assignment from the erv.json library works.
```
Comment (mdahlhausen) [inline test/necb/unit_tests/tests/test_necb_weather_hdd.rb:10]: comment seems to be for another test

File: test/necb/unit_tests/tests/test_necb_default_system_selection.rb_off
Comment (mdahlhausen) [inline test/necb/unit_tests/tests/test_necb_default_system_selection.rb_off:1]: what kind of file type is .rb_off?

File: openstudio-standards.gemspec [hunk 3]
```diff
@@ -48,6 +48,6 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'roo', '2.7.1'
   spec.add_development_dependency 'openstudio-api-stubs'
   spec.add_runtime_dependency 'tbd', '~> 3'
-  # spec.add_development_dependency 'aws-sdk-s3'
-  # spec.add_development_dependency 'git-revision'
+  #spec.add_development_dependency 'aws-sdk-s3'
```
Comment (mdahlhausen) [inline openstudio-standards.gemspec:51]: add space after # for comments

**Review by mdahlhausen — APPROVED** [review]
There are a few minor changes I commented on above. Otherwise looks fine.

I suggest you think about refactoring which regression tests you choose to run so that each merge doesn't involve changing every single regression model permutation and the test for it.

File: test/necb/unit_tests/tests/test_necb_default_system_selection.rb_off
Comment (ckirney) [reply test/necb/unit_tests/tests/test_necb_default_system_selection.rb_off:1]: I assume it is the type of file you don't want to use but don't want to get rid of either.  However, I don't really know what is going on with that file.  I'll ask around and see if I can delete it.

Comment (ckirney) [reply test/necb/unit_tests/tests/test_necb_default_system_selection.rb_off:1]: I will fix it but want to finish this merge first.  Are you okay if we keep it as is for now with the promise that I'll fix it before the next merge?

File: openstudio-standards.gemspec (hunk 3 above)
Comment (ckirney) [reply openstudio-standards.gemspec]: Sorry, fixed.

**Pull request merged**